### PR TITLE
anything beyond the most basic introduction should be in the documentation to avoid redundancy

### LINKED ISSUE #47: Make skeleton installable with composer create-project
Leverage `composer create-project` and composer scripts to make installer package.

### PR DESCRIPTION
**Pull request by dbu**
fix #47

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Install
 -------
 
 ```bash
-$ composer install
+$ composer create-project sculpin/blog-skeleton my-blog
```
Comment (dbu) [inline README.md]: this requires the package to be registered with packagist. can one of the maintainers please do that?

**Comment by dbu** [comment]
i added some of the things i remove here in https://github.com/sculpin/sculpin.io/pull/119

**Comment by bellisk** [comment]
@dbu I noticed that in https://github.com/sculpin/sculpin-blog-skeleton/pull/63 I added more information to the section that you'd already stripped out, oops. Let's look at it quickly together and see how best to arrange it?

File: README.md (hunk 1 above)
Comment (dbu) [reply README.md]: #47 is fixed

**Comment by dbu** [comment]
@beryllium i rebased now that the webpack encore thing is merged, and added setup instructions for composer and yarn so its really a complete instructions. i also tested the steps and got a running site :tada:

File: README.md [hunk 2]
```diff
@@ -59,77 +73,12 @@ $ php vendor/bin/sculpin generate --watch --server
...
-    rsync -avze 'ssh -p 999' --delete output_prod/ user@yoursculpinsite.com:public_html
+Documentation
+-------------
 
-In fact, `publish.sh` is provided to get you started. If you plan on deploying to an
-Amazon S3 bucket, you can use `s3-publish.sh` alongside the `s3cmd` utility (must be
-installed separately).
+The skeleton provides you with useful configuration and some example data for
+a Sculpin installation. Check out the [Get Started page](https://sculpin.io/getstarted/)
+and have a look atht the [documentation](https://sculpin.io/documentation/).
```
Comment (bellisk) [inline README.md]: typo: should be 'at the'

**Pull request merged**